src/extension/geofence/geofence-server/src/main/java/org/geoserver/geofence/server/rest/RulesRestController.java

```diff
@@ -167,6 +171,8 @@ public JaxbRuleList count(
             @RequestParam(value = "instanceId", required = false) Long instanceId,
             @RequestParam(value = "instanceName", required = false) String instanceName,
             @RequestParam(value = "instanceAny", required = false) Boolean instanceDefault,
+            @RequestParam(value = "ipaddress", required = false) String ipaddress,
+            @RequestParam(value = "ipaddressAny", required = false) Boolean ipaddressDefault,
```

Comment:
Code should be easier to follow if the same name is used everywhere, here we have ipAddressAny becoming ipAddressDefault

Reply:
it follows the same naming convention of the other fields :|